flink-scala-shell/src/test/scala/org/apache/flink/api/scala/ScalaShellITCase.scala

```diff
@@ -51,7 +45,7 @@ class ScalaShellITCase extends TestLogger {
 
     val output: String = processInShell(input)
 
-    Assert.assertTrue(output.contains(
+    Assert.assertTrue(output, output.contains(
```

Comment:
Add `output` so that we can see the real output when the test fails, otherwise it is difficult to see why this kind of test fails.